### PR TITLE
OPC-149 config to allow 5x combo pub list to pull from 1x

### PR DESCRIPTION
**Pull request by karendolan**
OPC-149 config to allow 5x combo pub list to pull from 1x engage server.

Includes documenting the params in otherpubs config template, adding an IGNORE-ME default value to alert otherpubs service that the params have not been intentionally set.

Note: there is an almost identical pull for the 1x to pull from 5x.

### CODE REVIEW

File: templates/default/edu.harvard.dce.otherpubs.OtherPubsServiceImpl.cfg.erb [hunk 1]
```diff
@@ -1,30 +1,62 @@
...
-# Enable Access to Legacy SD publications
+# For other (Legacy and other Opencast) course publication metadatadata, default course data, system message
+# Enable Access to Legacy SD publications and system message
 # Used by Combo publication listing page
 
+# I. OPC-149 combo page with other opencast engage search (i.e. get 5x pubs if 1x, get 1x pubs if 5x)
+#
+# I.1. Uncomment the following "oc.dns" param line add a different Opencast ENGAGE dns (search service endpoint)
+#    to facilitate pulling in pubs from the other Engage onto this Engage's combo pub list.
+#    Note: Only local pub is shown when other system pub has an overlapping date. Otherwise, both are shown.
```
Comment (rute-santos) [inline templates/default/edu.harvard.dce.otherpubs.OtherPubsServiceImpl.cfg.erb]: date and approximate time, correct?

Comment (karendolan) [reply templates/default/edu.harvard.dce.otherpubs.OtherPubsServiceImpl.cfg.erb]: If dates overlap (start + duration), and both are vod or both are live (historical check):
This is the full match function: 
https://bitbucket.org/hudcede/matterhorn-dce-fork/pull-requests/414/opc-149-combined-otherpubs-publist-1x/diff#Lmodules/mh-harvard-dce-engage-ui/combo/combopublist/combined/components/combojs/dataTransformService.jsT137

**Comment by rute-santos** [comment]
Looks OK to me.

**Review by lbjay — APPROVED** [review]
lgtm too

**Pull request merged**